source/css/index.css

```diff
@@ -183,6 +200,37 @@
   background-color: #0b5ed7;
 }
 
+/*
+* About Page Section
+*/
+
+.about-section {
+  padding-top: 20px;
+}
+
+.about-section .about-section__text p {
+  width: 90%;
+  font-size: 18px;
```

Comment:
Please see bootstrap's utilities for font sizes

https://getbootstrap.com/docs/5.0/getting-started/rfs/

tl;dr `rem` is preferred over `px`.

Reply:
@fulldecent using Bootstrap's RFS would mean setting up SCSS which I think you mentioned was a constraint.
I could manually just use the rem values and play around with css clamp as well if necessary.